TestCases/user_defined_functions/lam_flatplate.cfg

```diff
@@ -0,0 +1,99 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Test custom objective function.                            %
+% File Version 7.2.1 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+SOLVER= NAVIER_STOKES
+KIND_TURB_MODEL= NONE
+RESTART_SOL= NO
+%
+% "COMBO" is the name and group of the output for the objective function
+% (regardless of definition).
+SCREEN_OUTPUT= INNER_ITER, RMS_DENSITY, RMS_ENERGY, LINSOL_RESIDUAL, FORCE_Z,\
+               SURFACE_MASSFLOW, SURFACE_TOTAL_TEMPERATURE, COMBO
+HISTORY_OUTPUT = ITER, AERO_COEFF, FLOW_COEFF, FLOW_COEFF_SURF, COMBO
+OBJECTIVE_FUNCTION= CUSTOM_OBJFUNC
+% Here we define how the custom objective is computed from other outputs. For 
+% example, force in the z direction (computed for all MARKER_MONITORING and part
+% of AERO_COEFF) plus the absolute value of massflow across the second surface
+% ([1]) in MARKER_ANALYZE, scaled by a factor of 1000. It is also possible to
+% use "per surface" values from MARKER_MONITORING (use the dry-run mode to see
+% the names of available outputs, e.g. SU2_CFD -d lam_flatplate.cfg).
+% For multizone problems the CUSTOM_OBJFUNC should be defined for each zone
+% individually (with the outputs of that zone), the total for the problem is
+% the sum over zones, see disc_adj_fsi/Airfoil_2d.
+CUSTOM_OBJFUNC= '1e3 * (FORCE_Z + fabs(SURFACE_MASSFLOW[1]))'
```

Comment:
:medal_sports: Absolutely amazing! Being able to specify boundary markers is great... of course bonus points would be available for being able to specify the marker-tag instead of a number.

Reply:
That is a bit tricky, having first to deduce if it should come from marker analyse or marker monitoring.
But the eventual user defined functions with mass flow averages and so on would use lists of marker names.